src/pages/profile/index.js

```diff
@@ -0,0 +1,42 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import Avatar from '../../components/avatar';
+import { get } from '../../service/apiClient';
+// import { request } from '../../service/apiClient';
+// import { get } from '../../service/apiClient';
```

Comment:
remove unused code

Reply:
resolved

Reply:
removed unused code and commented-out code